diff --git a/sktime/datasets/_fpp3_loaders.py b/sktime/datasets/_fpp3_loaders.py
index ad28ec6..072bfa4 100644
--- a/sktime/datasets/_fpp3_loaders.py
+++ b/sktime/datasets/_fpp3_loaders.py
@@ -374,39 +374,43 @@ def _load_fpp3(dataset, temp_folder=None, robust=True):
 
 
 def load_fpp3(dataset, temp_folder=None):
-    """Load a dataset from the fpp3 package.
-
-    Returns ``pd.DataFrame`` in one of the valid sktime :term:`mtype` formats,
-    depending on the dataset.
-
-    Valid datasets are listed in ``datasets.DATASET_NAMES_FPP3``.
-
-    Requires ``rdata`` and ``requests`` packages in the environment.
-
-    Parameters
-    ----------
-    dataset : str
-        The name of the dataset to load.
-        Valid values are listed in ``datasets.DATASET_NAMES_FPP3``.
-    temp_folder: str, optional
-        Location of temporary data folder for downloading and extracting the dataset.
-        Deleted if the operation is successful.
-
-    Returns
-    -------
-    y : pd.DataFrame
-        The loaded data.
-        The mtype format is  ``pd.DataFrame`` for single time series,
-        ``pd-multiindex`` for collections of time series,
-        and ``pd_multiindex_hier`` for hierarchical time series.
-
-    Raises
-    ------
-    ValueError
-        If the dataset is not known.
-    RuntimeError
-        If there is an error loading the dataset.
-    """
-    y = _load_fpp3(dataset, temp_folder=None, robust=True)
-
-    return y
+    try:
+        """Load a dataset from the fpp3 package.
+
+        Returns ``pd.DataFrame`` in one of the valid sktime :term:`mtype` formats,
+        depending on the dataset.
+
+        Valid datasets are listed in ``datasets.DATASET_NAMES_FPP3``.
+
+        Requires ``rdata`` and ``requests`` packages in the environment.
+
+        Parameters
+        ----------
+        dataset : str
+            The name of the dataset to load.
+            Valid values are listed in ``datasets.DATASET_NAMES_FPP3``.
+        temp_folder: str, optional
+            Location of temporary data folder for downloading and extracting the dataset.
+            Deleted if the operation is successful.
+
+        Returns
+        -------
+        y : pd.DataFrame
+            The loaded data.
+            The mtype format is  ``pd.DataFrame`` for single time series,
+            ``pd-multiindex`` for collections of time series,
+            and ``pd_multiindex_hier`` for hierarchical time series.
+
+        Raises
+        ------
+        ValueError
+            If the dataset is not known.
+        RuntimeError
+            If there is an error loading the dataset.
+        """
+        y = _load_fpp3(dataset, temp_folder=None, robust=True)
+
+        return y
+
+    except Exception:
+        raise
\ No newline at end of file
